tsdb/engine/tsm1/compact.go

```diff
@@ -301,7 +303,7 @@ func (c *Compactor) Compact(tsmFiles []string) ([]string, error) {
 		return nil, nil
 	}
 
-	tsm, err := NewTSMKeyIterator(trs...)
+	tsm, err := NewTSMKeyIterator(tsdb.DefaultMaxPointsPerBlock, trs...)
```

Comment:
I will continue to review this code, but I want to check that this doesn't mean that this iterator will have any problems working with files that were created with a different value for `DefaultMaxPointsPerBlock`.